### PR TITLE
Release: 1.7.5

### PR DESCRIPTION
**Pull request by UlisesGascon**
**Plan to release it on Oct 30**

### What's included in the `HISTORY.md`

  * deps: accepts@~1.3.8
    - Fix sorting encoding with extra parameters
    - deps: mime-types@~2.1.34
    - deps: negotiator@0.6.3
  * deps: bytes@3.1.2
    - Add petabyte (`pb`) support
    - Fix "thousandsSeparator" incorrecting formatting fractional part
    - Fix return value for un-parsable strings
  * deps: compressible@~2.0.18
    - Mark `font/ttf` as compressible
    - Remove compressible from `multipart/mixed`
    - deps: mime-db@'>= 1.43.0 < 2'
  * deps: safe-buffer@5.2.1

### What's Changed
* chore: add support for OSSF scorecard reporting by @inigomarquinez in https://github.com/expressjs/compression/pull/186
* ci: fix errors in ci github action for node 8 and 9 by @inigomarquinez in https://github.com/expressjs/compression/pull/187
* docs: fix spelling by @dijonkitchen in https://github.com/expressjs/compression/pull/174
* deps: bytes@3.1.2 by @bjohansebas in https://github.com/expressjs/compression/pull/192
* remove --bail option by @bjohansebas in https://github.com/expressjs/compression/pull/196
* build: Node.js 23.0 by @bjohansebas in https://github.com/expressjs/compression/pull/193
* Replace accepts with negotiator by @blakeembrey in https://github.com/expressjs/compression/pull/197
* docs: update history by @bjohansebas in https://github.com/expressjs/compression/pull/200

### New Contributors
* @inigomarquinez made their first contribution in https://github.com/expressjs/compression/pull/186
* @dijonkitchen made their first contribution in https://github.com/expressjs/compression/pull/174
* @bjohansebas made their first contribution in https://github.com/expressjs/compression/pull/192
* @blakeembrey made their first contribution in https://github.com/expressjs/compression/pull/197

**Full Changelog**: https://github.com/expressjs/compression/compare/1.7.4...master

### CODE REVIEW

**Review by bjohansebas — APPROVED** [review]
LGTM

**Comment by bjohansebas** [comment]
ping

**Comment by bjohansebas** [comment]
@UlisesGascon Please merge #200 before releasing this version to keep the history synchronized

File: HISTORY.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-unreleased
+1.7.5 / 2024-10-21
```
Comment (wesleytodd) [inline HISTORY.md:1]: Remember to update this when we do publish.

**Review by wesleytodd — APPROVED** [review]

File: HISTORY.md (hunk 1 above)
Comment (UlisesGascon) [reply HISTORY.md:1]: Oh yeah!

**Comment by UlisesGascon** [comment]
I will target this for Monday, I prefer to avoid doing releases on the weekend :+1:

File: HISTORY.md (hunk 1 above)
Comment (UlisesGascon) [reply HISTORY.md]: Updated!

**Comment by UlisesGascon** [comment]
I updated the release date to today

**Pull request merged**